salt/modules/status.py

```diff
@@ -1054,8 +1054,12 @@ def master(master=None, connected=True):
     if not master_ips:
         return
 
+    pub_port = None
```

Comment:
Ah, OK. I don't think this is the right way to fix that then. We need to find out why `__salt__` isn't present in `salt.utils.minion` and fix it that way. The approach in this PR will only wallpaper over that problem and it will pop up again somewhere else, I suspect.